src/operator/numpy/np_flip_op.cc

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Move the code to the existing `np_matrix_op*` files.

Reply:
Done.